src/classes/TDTM_DMLgt200_TEST.cls

```diff
@@ -0,0 +1,465 @@
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+ * @author Salesforce.org
+ * @date 2017
+ * @group TDTM
+ * @group-content ../../ApexDocContent/TDTM.htm
+ * @description Test class to validate behavior changes related to TDTM Static Flag handling
+ *
+***/
+@isTest
+private class TDTM_DMLgt200_TEST {
+
+    /*********************************************************************************************************
+    * @description if you only want to run one test in this class, fill in its name here.
+    * if you want to run all tests, then use '*'
+    */
+    private static string strTestOnly = '*';
+
+    /* @description The number of records to insert for these tests. Inserting more than 200 records at once
+    * will cause triggers to be called twice - once for the first group of 200 and then again for the remainder.
+    */
+    private static Integer insertCnt = 210  ;
+
+    /*********************************************************************************************************
+    * @description Test creating 200+ Contacts with Affiliations
+    *   NOTE: THIS WILL FAIL IN AN ORG WITHOUT THE STATIC FLAG CHANGES
+    */
+    @isTest
+    static void test_contacts_with_affiliations() {
+        if (strTestOnly != '*' && strTestOnly != 'test_contacts_with_affiliations') return;
+
+        npe5__Affiliations_Settings__c affiliationsSettingsForTests = UTIL_CustomSettingsFacade.getAffiliationsSettingsForTests(
+                new npe5__Affiliations_Settings__c(npe5__Automatic_Affiliation_Creation_Turned_On__c = true));
+
+        Id orgRTId = [SELECT Id FROM RecordType WHERE SobjectType = 'Account' AND DeveloperName = 'Organization' AND IsActive = True LIMIT 1].Id;
+        Account orgAcct = new Account(Name='TestOrganization', RecordTypeId = orgRTId);
+        insert orgAcct;
+
+        Test.startTest();
+
+        // Create more than 200 Contact records and then validate that all of the Household Accounts were created
+        List<Contact> contacts = new List<Contact>();
+        for (Integer n=0; n<insertCnt; n++) {
+            contacts.add(new Contact(
+                    FirstName = 'TEST' + n,
+                    LastName = 'TestContact# ' + n,
+                    Title = 'TestTitle'
+            ));
+        }
+        database.insert(contacts);
+        system.AssertEquals(insertCnt, [SELECT Count() FROM Account WHERE RecordType.DeveloperName = 'HH_Account'],
+                'There should be exactly ' + insertCnt.format() + ' Household Account records');
```

Comment:
you can't assume the org has this record type.  it is not part of the package (though we include it in the trial template).